lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -453,6 +453,16 @@ def configure(conf)
       @resource.labels.freeze
       @common_labels.freeze
 
+      if @use_grpc
+        @construct_log_entry = method(:construct_log_entry_in_grpc_format)
+        @construct_log_request = method(:construct_log_request_in_grpc_format)
```

Comment:
Did you see my proposal of treating requests as hashes? We may still want the `@construct_log_entry` refactoring, but I think constructing the request and then decomposing it is overkill.

Reply:
Ah, saw it. I've changed it to use a hash instead.